apis/frontend/search-js/README.md

```diff
@@ -33,10 +33,85 @@ const response = await search({
     }
 }, {
     track: 'serp',
+    hitDecorators: [
+        priceDecorator(),
+        thumbnailDecorator({ size: "9" })
+    ],
+    maxRetries: 3,
+    retryInterval: 2000,
+    usePersistentCache: true,
+    useMemoryCache: false
+})
+
+```
+
+## Search Options
+
+The search function accepts all standard Nosto JS API options plus the following search-js specific options:
+
+### `hitDecorators`
+**Type:** `HitDecorator[]`  
+**Default:** `undefined`
+
+Hit decorators to apply to the search results. These allow you to transform search results with additional functionality like currency formatting and thumbnail generation.
+
+```ts
+import { priceDecorator } from "@nosto/search-js/currencies"
+import { thumbnailDecorator } from "@nosto/search-js/thumbnails"
+
+const response = await search(searchQuery, {
     hitDecorators: [
         priceDecorator(),
         thumbnailDecorator({ size: "9" })
     ]
 })
+```
+
+### `maxRetries`
+**Type:** `number`  
+**Default:** `0`
 
+Maximum number of retry attempts when a search request fails. Setting this to `0` disables retry logic.
+
+```ts
+const response = await search(searchQuery, {
+    maxRetries: 3 // Will retry up to 3 times on failure
+})
+```
+
+### `retryInterval`
+**Type:** `number`  
+**Default:** `1000`
+
+Interval (in milliseconds) between retry attempts when a search request fails.
+
+```ts
+const response = await search(searchQuery, {
+    maxRetries: 3,
+    retryInterval: 2000 // Wait 2 seconds between retries
+})
+```
+
+### `usePersistentCache`
+**Type:** `boolean`  
+**Default:** `false`
+
+Whether to use a persistent cache for the search results. When enabled, search results are cached across browser sessions.
+
```

Comment:
mention that only a single result will be cached and that this feature is primarily meant to support efficient back navigation from product pages to search results